python/pythonlab/unittest_runner/unittest_runner/validation_protocol.py

```diff
@@ -0,0 +1,29 @@
+from neighborhood import World, NeighborhoodContextType
```

Comment:
This isn't a wholesale new file; I had named the file with a typo ("procotol" instead of "protocol"). See the deleted file above for the previous version.